### DISCUSSION TITLE
pip install git+https://github.com/OpenMined/PySyft@dev#egg=syft  Will this take more time to get done?

### DISCUSSION THREAD

**Discussion by mpfouziya** [OpenMined/courses#307] [Course: Foundations of Private Computation]
pip install git+https://github.com/OpenMined/PySyft@dev#egg=syft is not proceeding forward... it is stuck...

Cloning https://github.com/OpenMined/PySyft (to revision dev) to c:\users\mpfou\appdata\local\temp\pip-install-2v7668do\syft...

**Comment by znreza** [comment]
Hi @mpfouziya have to got past this error? Since the newest version of PySyft is released few days back, you don't need to install pysyft from dev branch anymore. Please check it [here](https://github.com/OpenMined/PySyft/tree/0.5.0rc1) if you haven't yet.

**Reply by pjoshireply** [reply]
Hi @znreza,
I am running around in circles because of the same issue. I tried all other methods in the discussions:

- Install standard syft as in your link
- Install syft as @mpfouziya tried to do (following the most recent documentation in the course)
- Install syft as you suggested in discussion #295 (here I get a "Invalid requirement" error)

Are there any fixes to this?

Edit:
After noting the changes in the package path, the following seems to work:
`pip install 'git+https://github.com/OpenMined/PySyft@dev#egg=syft&subdirectory=packages/syft'`

P.S. : Use the link within double quotes in windows systems.

**Reply by GPradeepyaReddy** [reply]
The repository you have given works thank you so much!!
